src/figuresClasses.ts

```diff
@@ -1,19 +1,70 @@
 export interface Figure {
-
+  shape: string,
```

Comment:
```suggestion
  shape: Shape,
```
`type Shape = 'triangle' | 'circle' | 'rectangle'`